### PR TITLE
K.K. Slider songs

### PR DESCRIPTION
**Pull request by sh1989**

### CODE REVIEW

File: src/data/lists/ac-nh.json [hunk 1]
```diff
@@ -1583,6 +1583,304 @@
...
+        },
+        {
+          "value": "K.K. Ballad"
+        },
+        {
+          "value": "K.K. Bazaar"
+        },
+        {
+          "value": "K.K. Birthday",
+          "help": "Cannot be purchased"
```
Comment (McInkay) [inline src/data/lists/ac-nh.json:1649]: How do you get this one then? Is it on your birthday?

Comment (sh1989) [reply src/data/lists/ac-nh.json:1649]: It is, as I'm sure I'll find out soon for realsies.

Comment (McInkay) [reply src/data/lists/ac-nh.json:1649]: Alrighty. I'll update this when we get confirmation of exactly what we need to do.

Comment (sh1989) [reply src/data/lists/ac-nh.json:1649]: Thanks, the information out there is a bit confused - either you get it on your birthday or the following Saturday

**Pull request merged**